### ISSUE TITLE
State class of this entity " is not supported

### ISSUE THREAD

**Issue by zoopster** [dotKrad/hass-fpl#35]
Updated with latest commits as of friday 26 Aug...
When I look in developer tools under statistics the error shows from the "recorder" source.
State class  of this entity " is not supported
The sensors seem to work fine, but HASS wants to delete data associated with them.

Did the state class change or is this a regression with the latest changes?

**Comment by adamoutler** [comment]
Yes.  State classes were all commented out. Because of this, the sensors cannot be used with energy features. See here for more https://github.com/dotKrad/hass-fpl/blob/master/custom_components/fpl/sensor_DailyUsageSensor.py#L69

**Comment by adamoutler** [comment]
I didn't bother with a pull request because it appears a major refactoring its ongoing.

**Comment by zoopster** [comment]
Closing for activity
